libraries-bom-table-generation/updateREADMETable.py

```diff
@@ -63,7 +63,8 @@ def run_maven_flatten_clean():
                    'google-cloud-core-grpc',
                    'google-cloud-core-http',
                    'google-cloud-firestore-admin',
-                   'google-cloud-spanner-executor'
+                   'google-cloud-spanner-executor',
+                   'google-cloud-bigtable-stats' #exclude per https://github.com/googleapis/java-bigtable/blob/main/pom.xml#L334
```

Comment:
This HEAD pointer is likely to shift over time. 
Consider using a fixed tag: https://github.com/googleapis/java-bigtable/blob/v2.32.0/pom.xml#L334